src/P3_terminal_velocity.jl

```diff
@@ -51,24 +57,66 @@ function ice_particle_terminal_velocity(
     end
     v = sum(@. sum(ai * D^bi * exp(-ci * D)))
 
-    return ifelse(use_aspect_ratio, ϕᵢ(p3, D, F_r, th)^FT(1 / 3) * v, v)
+    return ifelse(use_aspect_ratio, ϕᵢ(p3, D, F_rim, th)^FT(1 / 3) * v, v)
+end
+
+"""
+   p3_particle_terminal_velocity(p3, D, Chen2022, ρₐ, F_rim, F_liq, th, use_aspect_ratio)
+
+ - p3 - p3 parameters
+ - D - maximum particle dimension
+ - Chen2022 - struct with terminal velocity parameters from Chen 2022
+ - ρₐ - air density
+ - F_rim - rime mass fraction (L_rim/(L_ice - L_liq)) [-]
+ - F_liq - liquid fraction (L_liq/L_ice) [-]
+ - th - thresholds as calculated by thresholds()
+ - use_aspect_ratio - Bool flag set to true if we want to consider the effects
+   of particle aspect ratio on its terminal velocity (default: true)
+
+Returns the terminal velocity of a single mixed-phase particle using the Chen 2022
+parametrizations by computing an F_liq-weighted average of solid and liquid
+phase terminal velocities, using the maximum dimension of the whole particle for both.
+"""
+function p3_particle_terminal_velocity(
+    p3::PSP3,
+    D::FT,
+    Chen2022::CMP.Chen2022VelType,
+    ρₐ::FT,
+    F_rim::FT,
+    F_liq::FT,
+    th,
+    use_aspect_ratio = true,
+) where {FT}
+    v_i = ice_particle_terminal_velocity(
+        p3,
+        D,
+        Chen2022.snow_ice,
+        ρₐ,
+        F_rim,
+        th,
+        use_aspect_ratio,
+    )
+    v_r = CM2.rain_particle_terminal_velocity(D, Chen2022.rain, ρₐ)
+    v = (1 - F_liq) * v_i + F_liq * v_r
```

Comment:
This `F_liq` weighing shows up in many places through the source code. I was wondering if it would be cleaner to add some sort of helper function to do this and then just call it throughout the code when needed.

Reply:
Ooh, you're totally right, that would be nice—I'll get on that with my changes